apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/TransactionPool.cs

```diff
@@ -243,40 +245,47 @@ public void MarkTransactionUsed()
                 }
             }
 
-            public async Task CreateTransactionAsync(Session session, TransactionOptions options)
+            public Task CreateTransactionAsync(Session session, TransactionOptions options)
```

Comment:
another note:  there was a race condition fixed here found during stress testing where the prewarm task was being kicked off when a session entered the pool, but the task variable "PreWarmTask" got assigned asynchronously -- this needs to be assigned synchronously.  

Or in general terms, whenever a session enters the pool, it must be in a state that can be consumed immediately by another task.  This means PreWarmTask state must be set before its added to the pool.